### ISSUE TITLE
Restore "Loans" as Transactions and Simplify Language

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/arctos#4902]
Request to simplify transaction terminology in the transaction dropdown- Can we come up with better options than "part-linked vs record-linked loan-like transactions"? How about just "Loans" - with options to select different tools within a loans menu if necessary?

![Screenshot 2022-08-03 15 59 18](https://user-images.githubusercontent.com/14808196/182720273-57e3877b-cd47-4056-9723-7e52cf8ae872.png)

Priority: medium to high, based on my level of frustration, but may be lower for others. Thoughts?

**Comment by dustymc** [comment]
merge-->https://github.com/ArctosDB/arctos/issues/4189

Better verbiage is most welcome, but it's been made clear that "loan" is not acceptable.

**Comment by campmlc** [comment]
This is in Enter Data - Batch Tools - "Loan-Like Transactions" -and then we go to "record-linked loan-like transactions" and "part-linked loan-like transactions" ?????
It is already difficult enough to find where to bulkload loan items vs loans. I would argue that this is like hiding the right tool in the basement in behind a locked door that says "Beware of the Leopard".
I propose we put the bulkload loan items tools under the Transactions dropdown - so that people working with loans can find them in the same place as they enter loans, not 4 levels deep in a totally different dropdown with utterly confusing terminology.

**Comment by campmlc** [comment]
And "Loans"  should be the header - then we go to what type of loans. "Loan-like transactions" is an abomination.
